.gitignore

```diff
@@ -38,15 +38,14 @@ test-results
 samples/_static/js/*.js*
 samples/_static/js/*
 samples/*/public/app.*
-samples/*/todos
 samples/*/users
-todos/
-users/
+!samples/angular-todo/*/todos
 
 # Environment files
 .env
 .env.local
 .angular
 samples/**/.env.local
 samples/**/.env
-samples/angularjs-todo/.env.local
\ No newline at end of file
+todos/*
+users/*
```

Comment:
Non-blocking: you might want to configure your IDE to always add as blank line at the end of files.

Reply:
Will do, thanks for the suggestion